src/main/java/stsjorbsmod/powers/FearPower.java

```diff
@@ -44,7 +49,17 @@ public void atStartOfTurn() { // note: monster's turn, not player's
                     boolean isFunctionallyDead = m.isDying || m.isDead || m.isEscaping || (m.halfDead && m.id.equals(Darkling.ID));
 
                     // Checking intent is to cover for the "multiple FearPowers triggering on the same turn" case
-                    if (!isFunctionallyDead && m.intent != Intent.ESCAPE) { ++remainingMonsters; }
+                    if (!isFunctionallyDead && m.intent != Intent.ESCAPE) {
+                        ++remainingMonsters;
+                        // negative if monster to the left of player, positive if monster to the right of player
+                        monsterDisplacementsFromPlayer.add(m.drawX - AbstractDungeon.player.drawX);
+                    }
+                }
+                // flipHorizontal = false -> face right
+                // flipHorizontal = true  -> face left
+                if (monsterDisplacementsFromPlayer.stream().noneMatch(x -> x < 0 == AbstractDungeon.player.flipHorizontal)) {
+                    // If none of the remaining monsters are on the same side the player is facing, then turn
+                    AbstractDungeon.player.flipHorizontal = !AbstractDungeon.player.flipHorizontal;
```

Comment:
This probably also wants to also lose SurroundedPower in the spire + spear casse